internal/openpgputil/openpgp.go

```diff
@@ -0,0 +1,113 @@
+package openpgputil
```

Comment:
Can we please name this as "pgputil" instead?  It's shorter and nicer to read.

Reply:
Done